src/main/java/com/programmers/springbootbasic/domain/voucher/repository/VoucherFileRepository.java

```diff
@@ -0,0 +1,69 @@
+package com.programmers.springbootbasic.domain.voucher.repository;
+
+import com.programmers.springbootbasic.domain.voucher.entity.Voucher;
+import jakarta.annotation.PostConstruct;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Repository;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+@Slf4j
+@Profile("default")
+@Repository
+public class VoucherFileRepository implements VoucherRepository {
+
+    private final String FOLDER_PATH = System.getProperty("user.dir") + "/data";
+    private String filePath;
+
+    @Value("${repository.voucher.fileName}")
+    private String fileName;
+
+    @PostConstruct
+    private void init() {
+        this.filePath = FOLDER_PATH + "/" + fileName + ".ser";
```

Comment:
```suggestion
        this.filePath = FOLDER_PATH + File.separator + fileName + ".ser";
```

윈도우에서는 \를 경로 구분자로 사용하고, 대부분의 다른 OS에서는 /를 사용합니다.
이를 대응하기 위해 Java에서는 File.separator를 사용하여 현재 OS의 파일 경로 구분자를 얻을 수 있습니다.

Reply:
[0cc6df4](https://github.com/prgrms-be-devcourse/springboot-basic/pull/875/commits/0cc6df4f2c1142db2006563cc972f9df57a15de6) 에 수정사항 반영했습니다!